.github/dependabot.yml

```
@@ -11,10 +11,14 @@ updates:
       interval: daily
     labels:
       - "topic: infrastructure"
+    commit-message:
+      prefix: "[skip-changelog] "
   - package-ecosystem: gomod
     directory: /
     schedule:
       interval: daily
     open-pull-requests-limit: 10
     labels:
       - "topic: infrastructure"
+    commit-message:
+      prefix: "[skip-changelog] "
```

```suggestion
    commit-message:
      prefix: "[skip changelog] "
  - package-ecosystem: gomod
    directory: /
    schedule:
      interval: daily
    open-pull-requests-limit: 10
    labels:
      - "topic: infrastructure"
    commit-message:
      prefix: "[skip changelog] "
```

Although either is functionally valid, it is best to use the canonical prefix format specified in the contributor guide:

https://arduino.github.io/arduino-cli/dev/CONTRIBUTING/#additional-settings

> If your PR doesn't need to be included in the changelog, please start the commit message and PR title with the string **[skip changelog]**

---

done in https://github.com/arduino/arduino-cli/pull/2586/commits/73a923bd4fc73fc1ea224ff0258116ceb50d74b6